pkg/controller/networkpolicy/networkpolicy_controller_perf_test.go

```diff
@@ -127,6 +127,58 @@ func TestInitXLargeScaleWithSmallNamespaces(t *testing.T) {
 	testComputeNetworkPolicy(t, 10*time.Second, namespaces, networkPolicies, pods)
 }
 
+func TestInitXLargeScaleWithLargeNamespaces(t *testing.T) {
```

Comment:
Is there a big improvement (computation time) for this test case? I remember when I was experimenting with DDlog (https://drive.google.com/file/d/1V1fMQ7VQaFPp-AO1aXbYC-fPoKPwo0gC/view), there was one single-namespace test case (TestPerf1) for which the Antrea Controller performed very badly.

Reply:
I'm still working on verifying the performance impact. I found the existing two cases are not really improved and suspected it's was because in TestInitXLargeScaleWithSmallNamespaces there were only a few pods in each namespace and it has only namespace scoped group so number of potential pod candidates (4 pods) that will be processed for each group doesn't make a difference, while in TestInitXLargeScaleWithOneNamespace, there was actually only single group, and the group is typically only handled less than 10 times, so having a cache didn't make a difference too.
I added this case which should be closer to practice that there is a good number of Pods and different groups in each Namespace to verify how much the cache can help. The result is promising: the execution time was reduced from 14 ~ 15s to 5 ~ 6s. I can check TestPerf1 too.